src/ResourceManagement/OperationalInsights/Microsoft.Azure.Management.OperationalInsights/Generated/Models/SearchMetadata.cs

```diff
@@ -0,0 +1,167 @@
+        /// <param name="aggregatedValueField">The aggregated value
+        /// field.</param>
+        /// <param name="aggregatedGroupingFields">The aggregated grouping
+        /// fields.</param>
+        /// <param name="sum">The sum of all aggregates returned in the result
+        /// set.</param>
+        /// <param name="max">The max of all aggregates returned in the result
+        /// set.</param>
+        /// <param name="schema">The schema.</param>
+        public SearchMetadata(string searchId = default(string), string resultType = default(string), long? total = default(long?), long? top = default(long?), string id = default(string), System.Collections.Generic.IList<CoreSummary> coreSummaries = default(System.Collections.Generic.IList<CoreSummary>), string status = default(string), System.DateTime? startTime = default(System.DateTime?), System.DateTime? lastUpdated = default(System.DateTime?), string eTag = default(string), System.Collections.Generic.IList<SearchSort> sort = default(System.Collections.Generic.IList<SearchSort>), long? requestTime = default(long?), string aggregatedValueField = default(string), string aggregatedGroupingFields = default(string), long? sum = default(long?), long? max = default(long?), SearchMetadataSchema schema = default(SearchMetadataSchema))
+        {
+            SearchId = searchId;
+            ResultType = resultType;
+            Total = total;
+            Top = top;
+            Id = id;
+            CoreSummaries = coreSummaries;
+            Status = status;
+            StartTime = startTime;
+            LastUpdated = lastUpdated;
+            ETag = eTag;
+            Sort = sort;
+            RequestTime = requestTime;
+            AggregatedValueField = aggregatedValueField;
+            AggregatedGroupingFields = aggregatedGroupingFields;
+            Sum = sum;
+            Max = max;
+            Schema = schema;
+        }
+
+        /// <summary>
+        /// Gets or sets the request id of the search.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "RequestId")]
+        public string SearchId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the search result type.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "resultType")]
+        public string ResultType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the total number of search results.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "total")]
+        public long? Total { get; set; }
+
+        /// <summary>
+        /// Gets or sets the number of top search results.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "top")]
+        public long? Top { get; set; }
+
+        /// <summary>
+        /// Gets or sets the id of the search results request.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "id")]
+        public string Id { get; set; }
```

Comment:
Nope. This is metadata of the search results. Not a resource. The Id here doesn't mean resourceId.